bangazonapi/views/product.py

```diff
@@ -98,14 +98,17 @@ def create(self, request):
         new_product.category = product_category
 
         if "image_path" in request.data:
```

Comment:
If you have nested if's like this, try to find a way to combine them `if this and that:`